pkg/sidecarexec/os_config_test.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 func Test_TrustedCertsCreateConfig(t *testing.T) {
-	backup, certs, close, err := createCertTempFiles(t)
+	backup, certs, closeFunc, err := createCertTempFiles(t)
```

Comment:
nit: closeFunc -> closeFile
(Same for other places)

Reply:
Adopted.